Formula/p/pytorch.rb

```diff
@@ -4,8 +4,8 @@ class Pytorch < Formula
   desc "Tensors and dynamic neural networks"
   homepage "https://pytorch.org/"
   url "https://github.com/pytorch/pytorch.git",
-      tag:      "v2.1.0",
-      revision: "7bcf7da3a268b435777fe87c7794c382f444e86d"
+      tag:      "2.1.1",
```

Comment:
The correct tag is `2.1.1`.

```suggestion
      tag:      "v2.1.1",
```

Reply:
Thanks